WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m

```diff
@@ -33,6 +33,7 @@
 
 NSString *const WPLegacyEditorNavigationRestorationID = @"WPLegacyEditorNavigationRestorationID";
 NSString *const WPLegacyAbstractPostRestorationKey = @"WPLegacyAbstractPostRestorationKey";
+NSString *const WPAppAnalyticsEditorSourceValueLegacy = @"legacy";
```

Comment:
@nheagy this is the old Legacy Editor. Right now we've got the Legacy, Hybrid, and Aztec around the corner.

I honestly wouldn't spend much time on this class, since it'll be removed in the next couple months!